### ISSUE TITLE
Bottom layers obstructed by height range modifier

### ISSUE THREAD

**Issue by CuredPrusa** [supermerill/SuperSlicer#4285]
Version: 2.5.60.0
Build: SuperSlicer_2.5.60.0_2024-05-21
Operating System: Windows
System Architecture: 64 bit
Windows Version: Windows 10 (build 19045), 64-bit edition

When setting HRM at first layers, slicer doesn't generate first solid layers even though HRM settings don't interact with layers (in my case it's only temperature and speed).

**Comment by supermerill** [comment]
yes, it's a problem. There is already some issues on it like #3397
Problem is, surface tags are generated per region(modifier).
First, i'll merge with 2.7 (as they changed a bit the surface generation)
then i'll see if it's possible to fix it.
but it can be a big task.
